scripts/docker/Dockerfile.openmptarget

```diff
@@ -0,0 +1,66 @@
+ARG BASE=nvidia/cuda:10.1-devel
+FROM $BASE
+
+RUN apt-get update && apt-get install -y \
+        bc \
+        git \
+        wget \
+        ccache \
+        python3 \
+        libelf-dev \
+        gcc-6 g++-6 \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ARG CMAKE_VERSION=3.10.3
+ENV CMAKE_DIR=/opt/cmake
+RUN CMAKE_KEY=2D2CEF1034921684 && \
+    CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION} && \
+    CMAKE_SCRIPT=cmake-${CMAKE_VERSION}-Linux-x86_64.sh && \
+    CMAKE_SHA256=cmake-${CMAKE_VERSION}-SHA-256.txt && \
+    wget --quiet ${CMAKE_URL}/${CMAKE_SHA256} && \
+    wget --quiet ${CMAKE_URL}/${CMAKE_SHA256}.asc && \
+    wget --quiet ${CMAKE_URL}/${CMAKE_SCRIPT} && \
+    gpg --keyserver hkps.pool.sks-keyservers.net --recv-keys ${CMAKE_KEY} && \
+    gpg --verify ${CMAKE_SHA256}.asc ${CMAKE_SHA256} && \
+    grep ${CMAKE_SCRIPT} ${CMAKE_SHA256} | sha256sum --check && \
+    mkdir -p ${CMAKE_DIR} && \
+    sh ${CMAKE_SCRIPT} --skip-license --prefix=${CMAKE_DIR} && \
+    rm ${CMAKE_SCRIPT}
+ENV PATH=${CMAKE_DIR}/bin:$PATH
+
+ENV LLVM_DIR=/opt/llvm
+RUN LLVM_VERSION=9.0.1 && \
+    LLVM_KEY=86419D8A && \
+    LLVM_URL=https://github.com/llvm/llvm-project/releases/download/llvmorg-${LLVM_VERSION} &&\
+    LLVM_ARCHIVE=llvm-project-${LLVM_VERSION}.tar.xz &&\
+    SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
+    wget --quiet ${LLVM_URL}/${LLVM_ARCHIVE} && \
+    wget --quiet ${LLVM_URL}/${LLVM_ARCHIVE}.sig && \
+    gpg --keyserver hkps.pool.sks-keyservers.net --recv-keys ${LLVM_KEY} && \
+    gpg --verify ${LLVM_ARCHIVE}.sig ${LLVM_ARCHIVE} && \
+    mkdir llvm-project && \
+    tar -xf ${LLVM_ARCHIVE} -C llvm-project --strip-components=1 && \
+    cd llvm-project && \
+    mkdir build && cd build && \
+    cmake \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX=$LLVM_DIR \
+      -DCMAKE_C_COMPILER=gcc-6 \
+      -DCMAKE_CXX_COMPILER=g++-6 \
```

Comment:
Don't use gcc to compile OpenMP target with clang. From the LLVM OpenMP mailing list
```
For the record, that will not work for the device runtime. It has to be
compiled by Clang in the near future (maybe already now). This is not
only for the performance benefits we get when we link it into the
application and remove all the clutter we do actually not need but also
because we are working towards a native OpenMP device runtime. 
```

Reply:
This is a 2-step approach.  I re-install OpenMP with clang after that.